node/src/components/consensus/tests/consensus_des_testing.rs

```diff
@@ -62,7 +62,7 @@ impl Display for ValidatorId {
 
 #[derive(Debug, Clone, Copy)]
 pub(crate) enum Fault {
-    Mute,
+    Mute(Timestamp, Timestamp),
```

Comment:
Can you refactor it to a proper struct with fields and use meaningful names (possible comments).

Reply:
Actually, how do you feel about adding a separate variant? Something like `TemporarilyMute`, `PermanentlyMute`? I imagine it would be difficult to figure out the `end_mute` timestamp if I wanted to have the validator be mute/offline permanently.